clients/go/zbc/oauthCredentialsProvider.go

```diff
@@ -173,16 +173,16 @@ func (provider *OAuthCredentialsProvider) getCachedCredentials() *OAuthCredentia
 }
 
 func applyCredentialEnvOverrides(config *OAuthProviderConfig) {
-	if envClientID := systemEnv.getenv(OAuthClientIdEnvVar); envClientID != "" {
+	if envClientID := systemEnv.get(OAuthClientIdEnvVar); envClientID != "" {
```

Comment:
Random question, what's the normal expected behavior if I have the following env:

```
ZEEBE_CERTIFICATE_PATH=
```

i.e. when the value is explicitly ""

I'm actually not sure, maybe you thought about it? :sweat_smile: 

I guess right now it's our way of "unsetting" the env var.

Reply:
Yes, we don't distinguish between an empty string and an unset variable except for the boolean plaintext connection var

Reply:
Could that be confusing for users? :thinking: 

If I'm a user, for `ZEEBE_PLAIN_TEXT` (or whatever it is), if I set it to empty it means false (i.e. it's used), but for the others it means not used. I'm leaning towards it being a little bit confusing, but also once you know it's probably one of those things you just roll with it. What do you think? If you were a user, I mean.

Reply:
I understand what you mean, it's not super intuitive. However, the advantage to setting it to insecure when "true" and to secure with any other value, you decrease the likelihood of setting it to "tru" (or some other misconfiguration) and sending a plaintext access token by mistake.